### ISSUE TITLE
[management] [portal] image media uri is not right

### ISSUE THREAD

**Issue by fabMrc** [gravitee-io/issues#5244]
In portal documentation I upload images
but there are no displayed
the html references the link : `/management/organizations/DEFAULT/environments/%7B:envId%7D/portal/media/F81216E93254ED9A4688E039D1382805`
of course it is not a portal url it is a management url and envId is not filled
If a replace it with `/portal/environments/DEFAULT/media/F81216E93254ED9A4688E039D1382805` it works
in the management area the WYSIWIG editor does not display the image also
in my assets/config.json I have
```
{
  "baseURL": "/portal/environments/DEFAULT"
}
```
And for management ui : 
```
{
  "baseURL": "/management/organizations/DEFAULT/environments/DEFAULT/"
}
```
When I edit documentation page the url is : `https://manage-api-rcc.foo.com/#!/environments/DEFAULT/settings/pages/db613db6-0097-4d3b-a13d-b600973d3b6a?tab=content&type=MARKDOWN`

**Comment by phiz71** [comment]
Hello @fabMrc, what is the version of your APIM installation?

**Comment by fabMrc** [comment]
3.6.0 exactly

**Comment by phiz71** [comment]
Ok thanks for the information.
I can reproduce your problem. I think this can be fixed in the 3.6.1, which should be available in a few days.

**Comment by fabMrc** [comment]
Oh yes ^^

**Comment by phiz71** [comment]
Actually, the bug already exists in 3.5 version. So it will be fixed in a 3.5.x version and then merged on 3.6 branch and on master.
